src/stories/pages/Login/Login.js

```diff
@@ -0,0 +1,134 @@
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import UserServices from '../../../firebase/CRUDServices/userServices';
+import { SplitGradient } from '../../layouts/SplitGradient/SplitGradient';
+import { Header } from '../../layouts/Header/Header';
+import { Footer } from '../../layouts/Footer/Footer';
+import { Branding } from '../../components/Branding/Branding';
+import { Main } from '../../layouts/Main/Main';
+import { Section } from '../../components/Section/Section';
+import { FormItem } from '../../components/FormItem/FormItem';
+import { Form } from '../../components/Form/Form';
+import { Button } from '../../components/Button/Button';
+
+/**
+ * Component for login page.
+ *
+ * @component
+ * @param {boolean} isDarkTheme theme color of the page.
+ * @return {object} (
+ *   <Login isDarkTheme={isDarkTheme}>
+ * )
+ */
+
+export const Login = ({ isDarkTheme }) => {
+  const [error, setError] = useState(false);
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const getUser = (e) => {
+    e.preventDefault();
+
+    const service = UserServices.getInstance();
+    console.log(service.getUsers()); // eslint-disable-line no-console
+
+    service.getUsers().then((users) => {
+      try {
+        users.forEach((item) => {
```

Comment:
Nitpicky but `find()` would make more sense to read. Also, shouldn't the user finding be handled in the backend?

Reply:
@cmutuc25 there's a `findUserById` in the BE, however, we don't have access to that when a user first logs onto the platform.

Reply:
Also, great suggestion! So much cleaner.